test/pkg/environment/aws/expectations.go

```diff
@@ -228,3 +279,43 @@ func (env *Environment) ExpectRunInstances(instanceInput *ec2.RunInstancesInput)
 
 	return out
 }
+
+func (env *Environment) ExpectSpotInterruptionRole() *iam.Role {
+	GinkgoHelper()
+	out, err := env.IAMAPI.GetRoleWithContext(env.Context, &iam.GetRoleInput{
+		RoleName: aws.String(fisRoleName),
+	})
+	Expect(err).ToNot(HaveOccurred())
+	return out.Role
+}
+
+func (env *Environment) ExpectAccountID() string {
+	GinkgoHelper()
+	identity, err := env.STSAPI.GetCallerIdentityWithContext(env.Context, &sts.GetCallerIdentityInput{})
+	Expect(err).ToNot(HaveOccurred())
+	return aws.StringValue(identity.Account)
+}
+
+func batchInstances(instanceIDs []string, size int) [][]string {
+	var instanceIDBatches [][]string
+	var currentBatch []string
+	for i, instanceID := range instanceIDs {
+		if i%size == 0 && len(currentBatch) > 0 {
+			instanceIDBatches = append(instanceIDBatches, currentBatch)
+			currentBatch = []string{}
+		}
+		currentBatch = append(currentBatch, instanceID)
+	}
+	if len(currentBatch) > 0 {
+		instanceIDBatches = append(instanceIDBatches, currentBatch)
+	}
+	return instanceIDBatches
+}
+
+func instanceIDsToARNs(instanceIDs []string, region string, accountID string) []string {
+	var arns []string
+	for _, instanceID := range instanceIDs {
+		arns = append(arns, fmt.Sprintf("arn:aws:ec2:%s:%s:instance/%s", region, accountID, instanceID))
+	}
```

Comment:
nit: same on lo.Map() here if you'd like

Reply:
Nice 🎉